lib/main_common.pm

```diff
@@ -399,6 +399,8 @@ sub load_boot_tests {
     }
     if ((get_var("UEFI") || is_jeos()) && !check_var("BACKEND", "svirt")) {
         loadtest "installation/data_integrity" if data_integrity_is_applicable;
+        # Enable jeos-firstboot, due to boo#1020019
+        loadtest "jeos/prepare_firstboot" if (is_jeos() && (is_arm || is_aarch64) && is_opensuse());
```

Comment:
Here, I would keep something like:
```
if ((is_arm || is_aarch64) && is_opensuse()) {
            # Enable jeos-firstboot, due to boo#1020019
            loadtest "installation/bootloader_uefi";
            loadtest "jeos/prepare_firstboot";
        }
```

Reply:
Frankly said, I find it safer to keep it in `load_jeos_tests`.

Reply:
Keeping it in `load_jeos_tests` is ok for me.

Reply:
i kept it in the `load_boot_tests` as it is everything together